conn.go

```diff
@@ -330,6 +336,7 @@ func (l *Conn) processMessages() {
 				_, err := l.conn.Write(buf)
 				if err != nil {
 					l.Debug.Printf("Error Sending Message: %s", err.Error())
+					message.Channel <- &PacketResponse{Error: fmt.Errorf("unable to send request: %s", err)}
```

Comment:
seems like this channel could get double-notified with this change... should we do this as well?
```
close(message.Channel)
delete(l.chanResults, message.MessageID)
```